### PR TITLE
Height of Binary Tree - Iterative

### PR DESCRIPTION
**Pull request by ankit-sr**
Added - Program for finding the height of a binary tree by iteration.
Updated - README
Issue - #4

### CODE REVIEW

File: GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+//Calling the create method and storing the address of the root node into root pointer declared above.    
+    root = node.create();
+    
+    cout<<"Tree created successfully : ";
+
+    cout<<"\nHeight of the tree is : "<<node.Height(root);
+    
+    return 0;
+}
```
Comment (imsushant12) [inline GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp]: Add one line and mentioon time+space complexity of your code.

File: GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+            if(node->left != NULL)
+                queue.push(node->left);
+
+            if(node->right != NULL)
+                queue.push(root->right);
+
+            nodeCount--;
+        }
+
```
Comment (imsushant12) [inline GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp]: remove extra lines.

Comment (imsushant12) [inline GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp]: remove extra lines.

File: GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            if(node->right != NULL)
+                queue.push(root->right);
+
+            nodeCount--;
+        }
+            
+
+    }
+
```
Comment (imsushant12) [inline GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp]: remove extra lines.

**Review by imsushant12 — CHANGES_REQUESTED** [review]

File: GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp (hunk 2 above)
Comment (ankit-sr) [reply GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp]: Removed

File: GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp (hunk 3 above)
Comment (ankit-sr) [reply GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp]: Removed

File: GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp (hunk 1 above)
Comment (ankit-sr) [reply GeeksForGeeks/C Plus Plus/Height_Binary_Tree_Iterative.cpp]: I've added both time and space complexities.

**Pull request merged**